paymentsheet/src/test/resources/payment-method-update.json

```
@@ -0,0 +1,48 @@
+{
+  "id": "[PAYMENT_METHOD_ID_HERE",
```

Looks like this is messed up. 
```suggestion
  "id": "[PAYMENT_METHOD_ID_HERE]",
```

Although not sure why we have square brackets here.